utils/src/config.rs

```diff
@@ -136,25 +119,24 @@ pub struct ServerTimeout {
     #[serde(with = "duration_format", default = "default_rpc_timeout")]
     rpc_timeout: Duration,
 
-    /// How long a candidate should wait before it starts another round of election
+    /// How many rounds of heartbeat_interval a candidate needs to wait until it starts a new round of election
+    /// It should be larger than `follower_timeout`
     #[getset(get = "pub")]
-    #[serde(with = "duration_format", default = "default_candidate_timeout")]
-    candidate_timeout: Duration,
+    #[serde(default = "default_candidate_timeout_round")]
+    candidate_timeout_round: u8,
```

Comment:
Just my two cents, `candidate_timeout_tick` is more clear.

Reply:
Fixed